smarts/core/utils/math.py

```diff
@@ -215,20 +224,40 @@ def low_pass_filter(
     lower_bound=-1,
     raw_value=0,
 ):
+    """Filters out large value jumps by taking a filter state and returning a filter state.
+    This is generally intended for filtering out high frequencies from raw signal values.
+    Args:
+        input_value: The raw signal value.
+        previous_filter_state: The last generated value from the filter.
+        filter_constant: The scale of the filter
+        time_step: The length of time between the previously processed signal and the current signal.
+        lower_bound: The lowest possible value allowed.
+        raw_value: A scalar addition to the signal value.
+    Returns:
+        The processed raw signal value.
+
+    """
     previous_filter_state += (
         time_step * filter_constant * (input_value - previous_filter_state)
     )
     previous_filter_state = np.clip(previous_filter_state + raw_value, lower_bound, 1)
     return previous_filter_state
 
 
+import math
+
+math.isclose
```

Comment:
What are these lines for?

```suggestion
```

Reply:
That was from attempting to check the description of `math.isclose` compared to our own `is_close`. I forgot to remove these lines.